changelog.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+### Refactor
+* [1853](https://github.com/zeta-chain/node/pull/1853) - refactor vote inbound tx and vote outbound tx
```

Comment:
Should be moved below

Reply:
Moved below to where? Are you referring to a line break after the heading?

Reply:
There is already a ### Refactor section below breaking changes